Amber serial build fails with new commit

Here's what I'm getting now:

```
clang++ -Wall -Wno-unused-function   -o ambpdb AmbPDB.o ActionFrameCounter.o ArgList.o Atom.o AtomMask.o BondSearch.o Box.o BufferedFrame.o BufferedLine.o ByteRoutines.o CharMask.o CIFfile.o CoordinateInfo.o CpptrajFile.o CpptrajStdio.o DistRoutines.o FileIO_Bzip2.o FileIO_Gzip.o FileIO_Std.o FileName.o FileTypes.o Frame.o InputTrajCommon.o MaskToken.o Matrix_3x3.o Mol2File.o NameType.o NetcdfFile.o OutputTrajCommon.o ParmFile.o Parm_Amber.o Parm_CIF.o Parm_CharmmPsf.o Parm_Gromacs.o Parm_Mol2.o Parm_PDB.o Parm_SDF.o Parm_Tinker.o PDBfile.o ProgressBar.o Range.o SDFfile.o StringRoutines.o TextFormat.o TinkerFile.o Timer.o Topology.o TrajectoryFile.o TrajFrameCounter.o Trajin_Single.o Trajout_Single.o Residue.o Traj_AmberCoord.o Traj_AmberRestart.o Traj_CIF.o Traj_Mol2File.o Traj_SDF.o Traj_AmberNetcdf.o Traj_Binpos.o Traj_Conflib.o Traj_SQM.o Traj_AmberRestartNC.o Traj_CharmmCor.o Traj_CharmmDcd.o Traj_GmxTrX.o Traj_Gro.o Traj_PDBfile.o Traj_Tinker.o Traj_NcEnsemble.o Vec3.o MpiRoutines.o \
               -L/home/swails/build_amber/amber/lib -lnetcdf -lz -lbz2
make[2]: Leaving directory '/home/swails/build_amber/amber/AmberTools/src/cpptraj/src/readline'
MpiRoutines.o: In function `printMPIerr':
MpiRoutines.c:(.text+0x1d): undefined reference to `MPI_Error_string'
MpiRoutines.c:(.text+0x2c): undefined reference to `MPI_Error_class'
MpiRoutines.o: In function `checkMPIerr':
[snip lots of undefined references]
make: *** [serial] Error 2
```

Still investigating...

MpiRoutines.o should not be a part of the build anymore. I'm away from my
comp for a bit but What I think might have happened is there is a reference
to MpiRoutines.c I forgot to remove.

On Monday, January 25, 2016, Jason Swails <notifications@github.com> wrote:

> Here's what I'm getting now:
>
> clang++ -Wall -Wno-unused-function   -o ambpdb AmbPDB.o ActionFrameCounter.o ArgList.o Atom.o AtomMask.o BondSearch.o Box.o BufferedFrame.o BufferedLine.o ByteRoutines.o CharMask.o CIFfile.o CoordinateInfo.o CpptrajFile.o CpptrajStdio.o DistRoutines.o FileIO_Bzip2.o FileIO_Gzip.o FileIO_Std.o FileName.o FileTypes.o Frame.o InputTrajCommon.o MaskToken.o Matrix_3x3.o Mol2File.o NameType.o NetcdfFile.o OutputTrajCommon.o ParmFile.o Parm_Amber.o Parm_CIF.o Parm_CharmmPsf.o Parm_Gromacs.o Parm_Mol2.o Parm_PDB.o Parm_SDF.o Parm_Tinker.o PDBfile.o ProgressBar.o Range.o SDFfile.o StringRoutines.o TextFormat.o TinkerFile.o Timer.o Topology.o TrajectoryFile.o TrajFrameCounter.o Trajin_Single.o Trajout_Single.o Residue.o Traj_AmberCoord.o Traj_AmberRestart.o Traj_CIF.o Traj_Mol2File.o Traj_SDF.o Traj_AmberNetcdf.o Traj_Binpos.o Traj_Conflib.o Traj_SQM.o Traj_AmberRestartNC.o Traj_CharmmCor.o Traj_CharmmDcd.o Traj_GmxTrX.o Traj_Gro.o Traj_PDBfile.o Traj_Tinker.o Traj_NcEnsembl
>  e.o Vec3
>  .o MpiRoutines.o \
>                -L/home/swails/build_amber/amber/lib -lnetcdf -lz -lbz2
> make[2]: Leaving directory '/home/swails/build_amber/amber/AmberTools/src/cpptraj/src/readline'
> MpiRoutines.o: In function `printMPIerr':
> MpiRoutines.c:(.text+0x1d): undefined reference to `MPI_Error_string'
> MpiRoutines.c:(.text+0x2c): undefined reference to `MPI_Error_class'
> MpiRoutines.o: In function `checkMPIerr':
> [snip lots of undefined references]
> make: *** [serial] Error 2
>
> Still investigating...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Amber-MD/cpptraj/issues/233>.
>


-- 
-------------------------
Daniel R. Roe, PhD
Department of Medicinal Chemistry
University of Utah
30 South 2000 East, Room 307
Salt Lake City, UT 84112-5820
http://home.chpc.utah.edu/~cheatham/
(801) 587-9652
(801) 585-6208 (Fax)

--- a/src/Makefile_at
+++ b/src/Makefile_at
@@ -18,7 +18,7 @@ OBJECTS=$(SOURCES:.cpp=.o) $(CSOURCES:.c=.o)
 
 include ambpdbfiles
 
-AMBPDB_OBJECTS=$(AMBPDBSOURCES:.cpp=.o) MpiRoutines.o
+AMBPDB_OBJECTS=$(AMBPDBSOURCES:.cpp=.o)
 
 all: cpptraj$(SFX)
 

